team-telnyx/webrtc
# fix(WEBRTC-149): Enabled multiple connections in multiple devices (browsers, tabs, and etc)

- Changed local storage to use session storage on the Login class to save the `sessid`

## 📝 To Do


## ✋ Manual testing

1. cd `packages/js` && `yarn install` && `yarn build`
2. navigate into `examples/vanilla`
3. open `index.html` in two tabs
4. register the same user in both tabs and see if they keep connected in both tabs.
5. Make a call and check if both tabs received the call.

## 🦊 Browser testing

### Desktop
